cmd/containerd-shim-runhcs-v1/task_hcs.go

```diff
@@ -859,3 +847,117 @@ func (ht *hcsTask) Stats(ctx context.Context) (*stats.Statistics, error) {
 	}
 	return s, nil
 }
+
+func (ht *hcsTask) Update(ctx context.Context, req *task.UpdateTaskRequest) error {
+	resources, err := typeurl.UnmarshalAny(req.Resources)
+	if err != nil {
+		return errors.Wrapf(err, "failed to unmarshal resources for container %s update request", req.ID)
+	}
+
+	if err := verifyTaskUpdateResourcesType(resources); err != nil {
+		return err
+	}
+
+	if ht.ownsHost && ht.host != nil {
```

Comment:
Assuming `ownsHost` will only be true for LCOW if this is the pause container task, do we not support updating the pause container resources? It seems we won't ever call into `updateTaskContainerResources` in that case.

Reply:
Should we support updating the pause container resources? What would the scenario be?